qpileup/test/org/qcmg/pileup/OptionsTest.java

```diff
@@ -1,8 +1,9 @@
-package org.qcmg.qpileup;
+package org.qcmg.pileup;
```

Comment:
any reason the package name is changing?

Reply:
I think there was typo, there are two folder under package org.qcmg for "qpileup",  but there is only one folder under other tools.  There is no reason have two different package names, they should be either "org.qcmg.qpileup" or "org.qcmg.pileup".

Reply:
sounds good, so have all tests now been moved from qpileup -> pileup?